damus/Views/PostView.swift

```diff
@@ -913,7 +909,7 @@ func build_post(state: DamusState, post: NSAttributedString, action: PostAction,
             }
     }
 
-    return NostrPost(content: content, kind: .text, tags: tags)
+    return NostrPost(content: content.trimmingCharacters(in: .whitespacesAndNewlines), kind: .text, tags: tags)
```

Comment:
Trimming is needed at the end because it's possible that the post is only uploaded media or the quoted note, but the user-inputted text is empty. We don't want the leading two newlines.